docs/pallas/grid_blockspec.md

```diff
@@ -33,7 +33,10 @@ for i in range(n):
 This generalizes to any tuple of integers (a length `d` grid will correspond
 to `d` nested loops).
 The kernel is executed as many times
-as `prod(grid)`. Each of these invocations is referred to as a "program".
+as `prod(grid)`.
+The default grid value `None` stands for `()`, and results in one
+kernel invocation.
+Each of these invocations is referred to as a "program".
```

Comment:
invocation -> instance?

Reply:
I tried to be consistent and the whole file uses "kernel invocation". Initially there was a mix of kernel invocation and iteration. Do you think that "instance" is better than "invocation" (to use throughout)?